candle-transformers/src/models/depth_anything_v2.rs

```diff
@@ -0,0 +1,535 @@
+use candle::D::Minus1;
+use candle::{Module, Result, Tensor};
+use candle_nn::ops::Identity;
+use candle_nn::{
+    batch_norm, conv2d, conv2d_no_bias, conv_transpose2d, linear, seq, Activation, BatchNorm,
+    BatchNormConfig, Conv2d, Conv2dConfig, ConvTranspose2dConfig, Sequential, VarBuilder,
+};
+
+use crate::models::dinov2::DinoVisionTransformer;
+
+pub struct ResidualConvUnit {
+    activation: Activation,
+    conv1: Conv2d,
+    conv2: Conv2d,
+    batch_norm1: Option<BatchNorm>,
+    batch_norm2: Option<BatchNorm>,
+}
+
+impl ResidualConvUnit {
+    pub fn new(
+        num_features: usize,
+        activation: Activation,
+        use_batch_norm: bool,
+        var_builder: VarBuilder,
```

Comment:
The usual "convention" in all other models is that the `VarBuilder` is named `vb` for shortness and we call `vb.pp` for `push_prefix`.